src/In.ProjectEKA.HipService/Startup.cs

```diff
@@ -182,5 +192,13 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             var consentContext = serviceScope.ServiceProvider.GetService<ConsentContext>();
             consentContext.Database.Migrate();
         }
+
+        private static bool CheckRoleInAccessToken(JwtSecurityToken accessToken)
+        {
+            var clientId = accessToken.Payload["clientId"] as string;
+            if (!(accessToken.Payload["resource_access"] is JObject resourceAccess) || clientId == null) return false;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5692149)